container_test.go

```diff
@@ -1,9 +1,11 @@
 package container_test
 
 import (
-	"github.com/golobby/container"
-	"github.com/stretchr/testify/assert"
 	"testing"
+
```

Comment:
do we need this empty lines?

Reply:
its standard format for Golang,
https://golang.org/cmd/gofmt/
https://blog.golang.org/go-fmt-your-code